index.html

```diff
@@ -37,5 +37,162 @@ <h4 class="connectmywork"> LET’S CONNECT</h4>
+            <button class="button1"> See Project</button>
+        </div>
+    </div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack_05">
+    <div class="card_05" >
+            <a href="#"><img src="./image/Snapshoot Portfolio1 (4).png" alt=""></a>
+    </div>
+    <div  class="multipost_05">
+        <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
+    </div>
+    <div class="work_main_5">
+        <ul class="works">
+            <li class="canopy"> CANOPY</li>
+            <li class="BackendDev">BackendDev</li>
+            <li class="num"> 2015 </li>
+        </ul>
+    
+    </div>
+    <div class="sectiondaily">
+        <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+    </div>
+    <div class="skil">
+        <ul class="program1">
+            <li class="Program-skilhtml">html</li>
+            <li class="Program-skilhtml">css</li>
+            <li class="skiljava">javascript</li>
+        </ul>
+    </div>
+   <div class="buttonsee">
+    <button class="button1"> See Project</button>
+</div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack2">
+    <div >
+        <h2  class="tonic4" >About Myself </h2>
+         <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+    </div>
+<div class = "social-section">
+        <div class="connectmywork">
+            <h4 > LET’S CONNECT</h4>
+        </div>
+        <div class="social">
+            <ul class="social-icons-mywork">
+                <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+            </ul>
+        </div>
+</div>
+        <div class="getme">
+            <button class="button1"> Get my resume </button>
+        </div>
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+              </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
```

Comment:
[OPTIONAL] I also don't think its a good idea to use images for the technologies. I suggest you get the language image and design this as a regular HTML